src/main/webapp/js/datahandlers/scholarshipdatahandler.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @fileoverview This class loads scholarship data from the servlet. */
+
+goog.module('datahandlers.scholarshipdatahandler');
+
+/** This class loads scholarship data from backend and format it for soy template. */
+class ScholarshipDataHandler {
+
+  /**
+   * @constructor
+   */
+  constructor() {
+    /** 
+     * This constant is the endpoint to send a fetch request to. 
+     * @private @const
+     */
+    this.SCHOLARSHIP_ENDPOINT_ = '/scholarship-data';
```

Comment:
Static variable doesn't compile for some reason.. I got a error says "=" is not allowed.

Reply:
I thought constants aren't supposed to be in the constructor. You could perhaps declare all of the constants at the top of the file, under the imports. This gives any instance of that class global access to the consts.

Reply:
I thought constants aren't supposed to be in the constructor. You could perhaps declare all of the constants at the top of the file, under the imports. This gives any instance of that class global access to the consts.

Reply:
Yeah "=" assignment is not supported with Google Closure compiler. I used a getter instead.